### PR TITLE
[KER-2024] Add DevOpsDays Kerala 2024 Event

### PR DESCRIPTION
**Pull request by libreminds**
-  [x] Add DevOpsDays Kerala 2024 to the event

### CODE REVIEW

File: data/events/2024/kerala/main.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+description: "Devopsdays is coming to Kerala!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted Kerala-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow Kerala-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:14]: Given that you have a signed venue agreement with a firm date, you should put that date here.

File: data/events/2024/kerala/main.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+name: "2024-kerala" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2024" # The year of the event. Make sure it is in quotes.
+city: "kerala" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopskerala" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:4]: This is not a valid [Xitter](https://x.com/devopskerala) account.

File: content/events/2024-kerala/sponsor.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Description = "Sponsor devopsdays Kerala 2024"
++++
+
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+
+<hr>
+
+devopsdays is a self-organizing conference for practitioners that depends on sponsorships. We do not have vendor booths, sell product presentations, or distribute attendee contact lists. Sponsors have the opportunity to have short elevator pitches during the program and will get recognition on the website and social media before, during and after the event. Sponsors are encouraged to represent themselves by actively participating and engaging with the attendees as peers. Any attendee also has the opportunity to demo products/projects as part of an open space session.
+<p>
+Gold sponsors get a full table and Silver sponsors a shared table where they can interact with those interested to come visit during breaks. All attendees are welcome to propose any subject they want during the open spaces, but this is a community-focused conference, so heavy marketing will probably work against you when trying to make a good impression on the attendees.
```
Comment (phrawzty) [inline content/events/2024-kerala/sponsor.md:13]: This is the boilerplate sponsor text. Is it true / accurate for _your_ event?

File: data/events/2024/kerala/main.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+# coordinates: "" # No longer used
+
+location: "Kerala" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set. Also used by the event_map shortcode!
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:35]: Given that you have a signed venue agreement with a firm date, you should indicate that somewhere. Here is a reasonable spot.

File: data/events/2024/kerala/main.yml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+ #- id: arresteddevops
+  # level: community
+
+sponsors_accepted : "yes" # Whether you want "Become a XXX Sponsor!" link
+
+# In this section, list the level of sponsorships and the label to use.
+# You may optionally include a "max" attribute to limit the number of sponsors per level. For
+# unlimited sponsors, omit the max attribute or set it to 0. If you want to prevent all
+# sponsorship for a specific level, it is best to remove the level.
+sponsor_levels:
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:84]: These are the boilerplate sponsor levels. Are they true for _your_ event?

**Review by phrawzty — CHANGES_REQUESTED** [review]
It looks like you ran the `add_new_event.sh` script then opened the PR, which is absolutely the first step—now you need to fill in details about the event in general.

File: data/events/2024/kerala/main.yml (hunk 4 above)
Comment (libreminds) [reply data/events/2024/kerala/main.yml]: Yes, added the same.

File: data/events/2024/kerala/main.yml (hunk 2 above)
Comment (libreminds) [reply data/events/2024/kerala/main.yml]: corrected it.. thanks !

File: data/events/2024/kerala/main.yml (hunk 1 above)
Comment (libreminds) [reply data/events/2024/kerala/main.yml]: Done. thanks!

File: content/events/2024-kerala/sponsor.md (hunk 3 above)
Comment (libreminds) [reply content/events/2024-kerala/sponsor.md]: I have corrected this to the initial plan we have in the refresh of this patch.. will add more details accordingly.

File: data/events/2024/kerala/main.yml (hunk 5 above)
Comment (libreminds) [reply data/events/2024/kerala/main.yml:84]: Corrected this as per the plan so far. thanks !

**Comment by libreminds** [comment]
> It looks like you ran the `add_new_event.sh` script then opened the PR, which is absolutely the first step—now you need to fill in details about the event in general.

Sure! added the requested details as a start for now. Will get these updated accordingly. Thanks!

File: data/events/2024/kerala/main.yml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+# coordinates: "" # No longer used
+
+location: "Kerala" # Defaults to city, but you can make it the venue name.
+#
+location_address: "Le Méridien Kochi, Nettoor, Maradu, Kochi, Ernakulam, Kerala 682304" #Optional - use the street address of your venue. This will show up on the welcome page if set. Also used by the event_map shortcode!
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:35]: Wow, the photos for this location look lovely! :D

File: data/events/2024/kerala/main.yml [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+description: "Devopsdays is coming to Kerala!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted Kerala-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow Kerala-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-09-28T08:00:00-08:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-09-28T20:00:00-08:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (phrawzty) [inline data/events/2024/kerala/main.yml:14]: Good job including the timestamp. 👍

**Review by phrawzty — APPROVED** [review]

File: data/events/2024/kerala/main.yml (hunk 6 above)
Comment (phrawzty) [reply data/events/2024/kerala/main.yml:35]: @libreminds Hah! I'd love to, but it's a bit of journey for me ;)

Comment (sanujan) [reply data/events/2024/kerala/main.yml:35]: @phrawzty Thanks for the quick turnaround, Dan.

**Pull request merged**